.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,37 @@
+name: Feature request
+description: Suggest a new idea for ILGPU
+title: "[FEATURE REQUEST]: <title>"
+labels: [ "feature" ]
+
+body:
+  - type: textarea
+    id: related-problem
+    attributes:
+      label: Is your feature request related to a problem? Please describe.
+      description: A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+    validations:
+      required: true
+
+  - type: textarea
+    id: describe-solution
+    attributes:
+      label: Describe the solution you'd like
```

Comment:
ISSUE: Inconsistent capitalization.

- Describe the solution you'd like
- Steps To Reproduce (in other templates)

We should be consistent.